### DISCUSSION TITLE
"log" ambiguous symbol when incliding imgui_internal.h

### DISCUSSION THREAD

**Discussion by zMidair** [ocornut/imgui#8154] [New Users Build/Link/Run issues ONLY!]
Hello!
I'm trying to include imgui_internal for dealing with dpi but I get this error. Does anyone know a possible solution?

This is the error I get:

![{4B7EA8DB-7D3D-4030-9E86-B7F77C94F847}](https://github.com/user-attachments/assets/c549e3d1-2309-4b88-9b36-9b27679e8013)

**Comment by ocornut** [comment]
Please provide more details.

**Reply by zMidair** [reply]
ImGui is conflicting with the ::log of another library I use. Is there a way to fix this?
